### PR TITLE
seqsero - request for review, new recipe & first contribution

### PR DESCRIPTION
**Pull request by estrain**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/seqsero/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package:
+  name: seqsero
+  version: 1.0
+
+source:
+  url: https://github.com/estrain/SeqSero/archive/1.0.tar.gz
+  sha256: d92a47b90c4b8b291a55258536e6c6c8ce2f34d5f597be2438e384f7340524d4 
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/seqsero/meta.yaml]: please start with 0

File: recipes/seqsero/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - samtools
+    - bwa
+    - blast
+
+  run:
+    - python
+    - sra-tools
+    - biopython 1.65
+    - scipy
+    - numpy 1.9.3
```
Comment (bgruening) [inline recipes/seqsero/meta.yaml]: Is it necessary to pin to a specific numpy and biopython version?

**Comment by mbargull** [comment]
@estrain: Even though I appreciate your effort to repackage SeqSero into a proper Python package in your fork, I'd rather like you to suggest that to the upstream repository first. Currently, the `setup.py` script in your fork has some issues, e.g., I believe it will not include all necessary files in the built package.

**Comment by mbargull** [comment]
That'll be great, thank you for doing this!
I'll close this PR then for now. Just re-open or open up another one when you're ready.

**Pull request closed without merging**